core_lib/src/tool/polylinetool.h

```diff
@@ -51,6 +52,7 @@ class PolylineTool : public StrokeTool
 
 private:
     QList<QPointF> mPoints;
+    bool mClosedOverride = false;
```

Comment:
Consider using enabled or on here to make it more explicit that we're dealing with a bool.

i'd also appreciate if we could name it something like *mClosedPathEnabled* instead, to make it less ambiguous.